chaos_genius/views/kpi_view.py

```diff
@@ -477,6 +479,8 @@ def edit_kpi(kpi_id):
             status = "failure"
     except Exception as err:  # noqa: B902
         status = "failure"
+        import traceback
+        logger.error(traceback.format_exc())
         logger.info(f"Error in updating the KPI: {err}")
```

Comment:
Could use logger's `exc_info` parameter instead:
```suggestion
        logger.info(f"Error in updating the KPI", exc_info=err)
```